gui.py

```
@@ -118,8 +118,18 @@ def __init__(self, q, master = None, **kwargs):
         self.pw_toggle_button.place(in_=self.pw_entry, relx=1.0, rely=0.5, x=5, anchor="w")
         self.login_button.pack(padx=20, pady=(10, 20))
 
+        try: 
+            self.iconbitmap(str(ASSETS_DIR / "icon.ico"))
+            # this try block is just so that it doesn't error on local runs with the source code or other OS's
+            # in reality, this should only affect the compiled version of windows by adding the icon to the taskbar
+            
+        except:
```

Ruff (0.15.10)

[error] Do not use bare `except`

(E722)